next.config.js

```diff
@@ -0,0 +1,6 @@
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  reactStrictMode: true,
+}
```

Comment:
Never disable strict mode, it's great for testing. You can look it up for more in detail explanations but basically it double renders everything to make sure no bugs can occur with weird timings. When building production it gets turned off by default